SS14.Admin/Pages/Bans/CreateMassBan.cshtml

```diff
@@ -0,0 +1,91 @@
+﻿@page
+@model SS14.Admin.Pages.Bans.CreateMassBanModel
+
+@{
+    ViewData["Title"] = "Create New Mass Ban";
+}
+
+<partial name="Shared/StatusMessageAlert" />
+
+
+    <div class="container">
+        <h1>Mass Ban</h1>
+
+        <form method="post" enctype="multipart/form-data" id="uploadForm">
+            <div class="form-group">
+
+                <div class="custom-file m-1">
+                    <label for="file">Choose a .TSV file to Upload:</label>
```

Comment:
You should probably explain the expected format somewhere. I.e. what columns and in which order and that you need a header row

Reply:
done